### PR TITLE
PLC4X-255 Fix for Kafka Source CPU Usage

### PR DESCRIPTION
**Pull request by hutcheb**

### CODE REVIEW

File: plc4j/integrations/apache-kafka/src/main/java/org/apache/plc4x/kafka/Plc4xSourceTask.java [hunk 1]
```diff
@@ -219,7 +220,13 @@ public void stop() {
             buffer.drainTo(result, numElements);
             return result;
         } else {
```
Comment (ottobackwards) [inline plc4j/integrations/apache-kafka/src/main/java/org/apache/plc4x/kafka/Plc4xSourceTask.java:234]: Is there some way to make these configurable?

Comment (hutcheb) [reply plc4j/integrations/apache-kafka/src/main/java/org/apache/plc4x/kafka/Plc4xSourceTask.java:234]: Well that took longer than I thought it would.

Comment (ottobackwards) [reply plc4j/integrations/apache-kafka/src/main/java/org/apache/plc4x/kafka/Plc4xSourceTask.java:234]: I don't know if there is any logging in this area of the code, but the poll() function would seem to be a good place to put some trace logging in around what we are doing there.
I am ok with that being some kind of follow on though

Comment (ottobackwards) [reply plc4j/integrations/apache-kafka/src/main/java/org/apache/plc4x/kafka/Plc4xSourceTask.java:234]: Actually there is almost no logging in here, not this PR's issue

**Review by ottobackwards — APPROVED** [review]
Thanks for the contribution and the extra work to make this configurable

**Pull request merged**